R/servers.R

```diff
@@ -123,30 +120,26 @@ discoverServers <- function(quiet = FALSE) {
   invisible(introduced)
 }
 
-getDefaultServer <- function(local = FALSE, prompt = TRUE) {
-   existing <- servers(local)
-   # if there are no existing servers, silently try to discover one to work with
-   if (length(existing) == 0 || nrow(existing) == 0) {
-     discoverServers(quiet = TRUE)
-     existing <- servers(local)
-   }
-
-   # if exactly one server exists, return it
-   if (nrow(existing) == 1) {
-     return(list(name = as.character(existing[, "name"]),
-                 url = as.character(existing[, "url"])))
-   }
-
-   # no default server, prompt if there are multiple choices
-  if (nrow(existing) > 1 && prompt && interactive()) {
-    name <- as.character(existing[1, "name"])
-    message("Registered servers: ", paste(existing[, "name"], collapse = ", "))
-    input <- readline(paste0(
-      "Which server (default '", name, "')? "))
-    if (nchar(input) > 0) {
-      name <- input
-    }
-    return(serverInfo(name))
+findServer <- function(prompt = TRUE) {
```

Comment:
Need to add `server` argument so can do argument checking like in `findAccount()`